src/main/webapp/WEB-INF/tags/instructor/results/expandCollapseButton.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="instructorFeedbackResultsTop - Expand Collapse Button" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="filterPanel" type="teammates.ui.template.InstructorFeedbackResultsFilterPanel" required="true" %>
```

Comment:
Instead of passing in the whole `filterPanel`, just pass in the `sortType` as a string attribute.